contracts/currency-network/CurrencyNetworkInterface.sol

```diff
@@ -23,10 +43,21 @@ interface CurrencyNetworkInterface {
     )
         external;
 
+    /**
+     * @notice returns what _b owes to _a
```

Comment:
Isn't it what `_from` owes to `_to`?

Reply:
No, as we can see in the implementation:
https://github.com/trustlines-protocol/contracts/blob/master/contracts/currency-network/CurrencyNetworkBasic.sol#L458